src/main/webapp/js/pages/savedPlanners.js

```diff
@@ -66,6 +83,31 @@ function createCard(planName, tableData, creditsData) {
   document.getElementById('plan-column').appendChild(card);
 }
 
+/**
+ * Tells the server to delete the plan.
+ * @param {Number} id The id of the plan to delete.
+ */
+async function deletePlan(id) {
+  const params = new URLSearchParams();
+  params.append('id', id);
+  let response;
+  try {
+    response =
+        await fetch('/api/planner/delete', {method: 'POST', body: params});
+  } catch (err) {
+    Util.createAlert(
+        'Could not delete element.', 'danger',
+        document.getElementById('plan-name'));
```

Comment:
I think you want to return here instead of falling through. Otherwise `response` will be undefined and you will probably hit an exception on line 102 when attempting access `undefined.ok`.

Reply:
i totally missed that, thanks